cmd/baton-crowdstrike/main.go

```diff
@@ -6,39 +6,50 @@ import (
 	"os"
 
 	"github.com/conductorone/baton-crowdstrike/pkg/connector"
-	"github.com/conductorone/baton-sdk/pkg/cli"
+	configSchema "github.com/conductorone/baton-sdk/pkg/config"
 	"github.com/conductorone/baton-sdk/pkg/connectorbuilder"
+	"github.com/conductorone/baton-sdk/pkg/field"
 	"github.com/conductorone/baton-sdk/pkg/types"
 	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
+	"github.com/spf13/viper"
 	"go.uber.org/zap"
 )
 
-var version = "dev"
+const (
+	version       = "dev"
+	connectorName = "baton-crowdstrike"
+)
+
+var (
+	clientId            = field.StringField(connector.ClientId, field.WithRequired(true), field.WithDescription("CrowdStrike client ID used to generate the access token."))
+	clientSecret        = field.StringField(connector.ClientSecret, field.WithRequired(true), field.WithDescription("CrowdStrike client secret used to generate the access token."))
+	region              = field.StringField(connector.Region, field.WithRequired(true), field.WithDescription("CrowdStrike region to connect to."))
```

Comment:
Can you update the description to include the available regions?
https://github.com/ConductorOne/baton-crowdstrike/blob/main/pkg/connector/connector.go#L93-L104
```go
region = field.StringField(
  connector.Region, 
  field.WithRequired(true), 
  field.WithDescription(
    "CrowdStrike region to connect to. Options include 'us-1', 'us-2', 'eu-1', and 'us-gov-1'" 
  ),
)
```

Reply:
Fixed